packages/wrangler/src/deployment-bundle/esbuild-plugins/hybrid-nodejs-compat.ts

```diff
@@ -90,14 +98,16 @@ function handleRequireCallsToNodeJSBuiltins(build: PluginBuild) {
 function handleUnenvAliasedPackages(
 	build: PluginBuild,
 	alias: Record<string, string>,
-	external: string[]
+	external: string[],
+	unenvPresetPath: string | undefined
 ) {
 	// esbuild expects alias paths to be absolute
 	const aliasAbsolute: Record<string, string> = {};
 	for (const [module, unresolvedAlias] of Object.entries(alias)) {
 		try {
+			const paths = unenvPresetPath ? [unenvPresetPath] : undefined;
```

Comment:
Can we just call 

```
const unenvPresetPath = getUnenvPresetPathFromEnv();
```

here and avoid having to thread it through the code?

Reply:
(quick call with Vic, we might eventually move the plugin to unenv core as part of https://github.com/unjs/unenv/issues/263 -- this unifies impl between nitro and wrangler as a bonus)

Reply:
Not sure I follow here.
If we move this to the preset (or even to unenv core) then this `unenvPresetPath` would be moot, right? So we'd need to redo this code then anyway.
In that case just loading the env var where it is needed is simplest and touches the least code.

Reply:
So there are two things:

- To move forward add a feature to unenv core to allow full path resolves
- In the future, to move the plugin stuff as an unplugin

Reply:
Does that affect my original comment to move the reading of the environment variable into this code?

Reply:
> If we move this to the preset (or even to unenv core) then this unenvPresetPath would be moot, right?

It would only be a different path but same code.

> In that case just loading the env var where it is needed is simplest and touches the least code.
> Does that affect my original comment to move the reading of the environment variable into this code?

In general I find injecting values cleaner.
i.e. in this case if we have an hardcoded `WRANGLER_...` name, we would have to change that if we move the plugin. Injecting the value is more flexible (i.e. the first point above).